.gitattributes

```diff
@@ -0,0 +1,6 @@
+/doc export-ignore
+/features export-ignore
+/tests export-ignore
```

Comment:
There is no `tests` folder but a `spec` folder

Reply:
fixed. copy&paste error